### PR TITLE
Create messages.it.yml

### PR DESCRIPTION
**Pull request by jamiroconca**
| Q             | A
| ------------- | ---
| Translation   | yes|no

baseline for italian translations (cloned from english)

### CODE REVIEW

**Comment by tcitworld** [comment]
Hi,
As written [here](http://doc.wallabag.org/en/master/developer/translate.html#configuration-file), you need to edit also the configuration file to enable the new translation.

**Comment by jamiroconca** [comment]
hi,
pull requests for config.yml and CraueConfigBundle.it.yml have been created

**Comment by nicosomb** [comment]
I added you in our translator team. If you want to leave this group, you can ;-)

**Comment by jamiroconca** [comment]
thank you...

File: src/Wallabag/CoreBundle/Resources/translations/messages.it.yml [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+                label: 'Operator'
+                less_than: 'Less than...'
+                strictly_less_than: 'Strictly less than...'
+                greater_than: 'Greater than...'
+                strictly_greater_than: 'Strictly greater than...'
+                equal_to: 'Equal to...'
+                not_equal_to: 'Not equal to...'
+                or: 'One rule OR another'
+                and: 'One rule AND another'
+                matches: 'Tests that a <i>subject</i> is matches a <i>search</i> (case-insensitive).<br />Example: <code>title matches "football"</code>'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.it.yml]: All these lines (from line 92 to line 128) aren't translated)

**Comment by j0k3r** [comment]
@jamiroconca feel free to provide other PR to finish the italian translation of this file.

**Pull request merged**